CRM/Contact/Task.php

```diff
@@ -182,6 +176,14 @@ public static function initTasks() {
         ),
       );
 
+      $maxEmailKillSwitch = Civi::settings()->get('max_emails_kill_switch');
+      self::$_tasks[self::EMAIL_CONTACTS] = array(
+        'title' => ts('Email - send now (to ' . $maxEmailKillSwitch . ' or less)'),
```

Comment:
You can't concatenate variables directly into the `ts()` function. Instead, do 

    ts('Email - send now (to %1 or less)', array(1 => $maxEmailKillSwitch))